db/data/20220720142258_change_newflow_social_to_facebook_google.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ChangeNewflowSocialToFacebookGoogle < ActiveRecord::Migration[5.2]
```

Comment:
We may need `disable_ddl_transaction!` in the class depending on how many records there are in prod. If there are too many, the transaction will cause issues due to locking a bunch of rows

Reply:
added